src/interface.js

```diff
@@ -0,0 +1,111 @@
+$(document).ready(function() {
+  var game = new Game();
+
+  $('#btn9').on('error', function(){
+    $("#errorbox").text("Don't CHEAT!!");
+  });
+
+  $('#totalscore').text(game._score);
+
+  $('#btn0').on('click', function(){
+    game.buttonActions(0);
+  });
+  $('#btn1').on('click', function(){
+    game.buttonActions(1);
+  });
+  $('#btn2').on('click', function(){
+    game.buttonActions(2);
+  });
+  $('#btn3').on('click', function(){
+    game.buttonActions(3);
+  });
+  $('#btn4').on('click', function(){
+    game.buttonActions(4);
+  });
+  $('#btn5').on('click', function(){
+    game.buttonActions(5);
+  });
+  $('#btn6').on('click', function(){
+    game.buttonActions(6);
+  });
+  $('#btn7').on('click', function(){
+    game.buttonActions(7);
+  });
+  $('#btn8').on('click', function(){
+    game.buttonActions(8);
+  });
+  $('#btn9').on('click', function(){
+    game.buttonActions(9);
+  });
+  $('#btn10').on('click', function(){
+    game.buttonActions(10);
+  });
+});
+
+Game.prototype.buttonActions = function(number) {
+  this.nextBowl(number);
+  $("#frame" + this._currentFrame + "roll" + this._currentRoll).text(this.activeFrame()._rolls[this._currentRoll-1]);
+  $("#framescore" + this._currentFrame).text(this.activeFrame()._frameScore);
+  $("#totalscore").text(this._score);
+  if(this._frames.length > 1) {
+    $("#framescore" + (this._currentFrame-1)).text(this.previousFrame()._frameScore);
+  }
+  if(this._frames.length > 2) {
+    $("#framescore" + (this._currentFrame-2)).text(this.twoFramesAgo()._frameScore);
+  }
+  this.hideNumbers(number);
+  this.resultMessage(number);
+  this.errorMessage();
+  this.gutterGameMessage();
+  this.perfectGameMessage();
+  this.updateCurrentFrame();
+};
+
+Game.prototype.hideNumbers = function(number) {
```

Comment:
This function's cyclomatic complexity is too high. (5)<br>'Game' is not defined.